packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -408,6 +411,12 @@ export class Slider extends FormAssociated<HTMLInputElement>
                     ? e.pageX - this.getBoundingClientRect().left
                     : e.pageY;
 
+            if (this.orientation === Orientation.vertical) {
```

Comment:
Will this same problem exist in horizontal-scrolling scenarios?

Reply:
I'm only calling it in mouseDown handler now and I'm doing it for both orientations